main.tf

```diff
@@ -40,6 +40,9 @@ resource "aws_db_instance" "main_rds_instance" {
 
     backup_retention_period = "${var.backup_retention_period}"
     backup_window = "${var.backup_window}"
+    tags {
+        "Name" = "Test"    
```

Comment:
@gstlt Same comment here, this tag does not provide any value.
@deepakagrawal2 Are you going to parameterize `Name` tag?